src/components/App.tsx

```diff
@@ -12,7 +12,7 @@ function App() {
         <Route path="/" element={<SharedLayout />}>
           <Route path="/" element={<HomePage />} />
           <Route path="/teachers" element={<TeachersPage />} />
-          <Route path="/favorites" element={<FavoritesPage />} />
+          <Route path="/favorites" element={<FavoritesPage teacher={[]} />} />
```

Comment:
```suggestion
          <Route path="/favorites" element={<FavoritesPage />} />
```
remove it, idk for what it is